lte/gateway/python/integ_tests/s1aptests/test_sctp_shutdown_while_mme_is_stopped.py

```diff
@@ -55,9 +55,10 @@ def test_sctp_shutdown_while_mme_is_stopped(self):
         self._s1ap_wrapper._s1_util.receive_emm_info()
 
         print("Stopping MME service")
-        self._s1ap_wrapper.magmad_util.exec_command(
-            "sudo service magma@mme stop",
-        )
+        self._s1ap_wrapper.magmad_util.disable_service('mme')
```

Comment:
For now its ok, but we can keep in mind that we can enhance these enable/disable functions later in future to take array of services because we have multiple service dependencies.